lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -29,33 +29,37 @@
           $mime_split = explode(':', $task->file);
           $task->file = $mime_split[0] . ':' . $mimetype . $mime_split[1];
         }
-        $values = [
+
+        $data = [
           'nid' => $task->campaign_id,
```

Comment:
hmmm, I think it makes it a little less clear if there's multiple ways this data can be formatted though